packages/ember/src/index.ts

```diff
@@ -139,11 +139,9 @@ module.exports = {
   // of the caching implicitly by specifying treeFor* methods
   cacheKeyForTree(treeType: string): string {
     switch (treeType) {
-      case 'app': {
-        const sources = (this.docfyConfig as DocfyConfig)?.sources
-          ?.map((item) => item.root)
-          ?.join(',');
-        return cacheKeyForTree(treeType, this, [sources]);
+      case 'app':
+      case 'addon': {
```

Comment:
without the addon entry, I'd get: 
![image](https://user-images.githubusercontent.com/199018/146609541-430f5151-b751-4ce1-b741-3369577a1924.png)